ForMaL/Homeworks/homework1.lean

```diff
@@ -0,0 +1,102 @@
+/-
+# Homework 1: Basics
+References: [MIL] Mathematics in Lean, [Tut] Tutorials project.
+-/
+
+import Mathlib.Tactic
+
+/-
+# Problem  1
+Read the following section of [Tut] and complete the exercises 0013, 0014, 0015. The exercise is to prove the same statement using backwards reasoning, forwards reasoning, and a combination of both.
+-/
+
+/-
+We announce the use of a lemma, and provide proofs after the fact, using
+the `apply` tactic. This is called backward reasoning because we get the conclusion
+first, and provide proofs later. Using `rw` on the goal (rather than on an assumption
+from the local context) is also backward reasoning.
```

Comment:
don't know why, but this reads super weird

Reply:
Yeah I also wasn't quite happy with it, it's lifted from the tutorials project but I can rephrase it.

Reply:
Have rephrased.